edgelet/support-bundle/src/support_bundle.rs

```diff
@@ -363,7 +363,7 @@ where
         /***
          * Note: this assumes using windows containers on a windows machine.
          * This is the expected production scenario.
-         * Since the bundle command does not read the config.yaml, it cannot use the `moby.runtime_uri` from there.
+         * Since the bundle command does not read the config.toml, it cannot use the `moby.runtime_uri` from there.
```

Comment:
Just delete this whole comment block and the line of code it applies to. We don't care about `cfg(windows)` any more.

Reply:
Fixed